samples/app-hello-world/lib/utils.py

```diff
@@ -0,0 +1,101 @@
+import urllib3
+import os
+import json
+import base64
+from urllib.parse import urlencode
+import time
+
+
+arbiter = os.environ['DATABOX_ARBITER_ENDPOINT']
+hostname = os.environ['DATABOX_LOCAL_NAME']
+
+arbiterToken = open("/run/secrets/ARBITER_TOKEN", "rb").read()
+arbiterTokenbase64 = base64.b64encode(arbiterToken)
+
+CM_HTTPS_CA_ROOT_CERT = open("/run/secrets/DATABOX_ROOT_CA").read()
+
```

Comment:
these `open(...).read()` calls might need some protection against files not existing?
(or is that a cast-iron guarantee from the CM?)